components/Alert/alert.css

```diff
@@ -0,0 +1,73 @@
+*{
+    border-width: 0px;
```

Comment:
You can use reset CSS, as it improves the expected behaviour. The way Vikram taught us.

Reply:
Can you explain me a bit?

Reply:
```
* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}
```

This is known as the basic reset code. It will remove the default margin, padding, and will set the box-sizing to border box.